diff --git a/templates/recipes/list.html.twig b/templates/recipes/list.html.twig
index bbf02b8..d5705bf 100644
--- a/templates/recipes/list.html.twig
+++ b/templates/recipes/list.html.twig
@@ -3,11 +3,6 @@
 {% block title %}All Recipes{% endblock %}
 
 {% block body %}
-    <div class="hero-wrapper">
-        <h1>Doggone Good Recipes</h1>
-        <p>Recipes your pup will love!</p>
-    </div>
-
     <div class="container">
         <div class="recipe-list-container py-5 px-3 mt-4" style="background-color: #fdedf0; border-radius: 5px;">
             {% for recipe in pager %}
