diff --git a/src/Layouts/VerticalWhitespacePlugin.php b/src/Layouts/VerticalWhitespacePlugin.php
index fcf273d..dce7bac 100644
--- a/src/Layouts/VerticalWhitespacePlugin.php
+++ b/src/Layouts/VerticalWhitespacePlugin.php
@@ -2,12 +2,13 @@
 
 namespace App\Layouts;
 
+use Netgen\Layouts\Block\BlockDefinition\BlockDefinitionHandlerInterface;
 use Netgen\Layouts\Block\BlockDefinition\Handler\Plugin;
 
 class VerticalWhitespacePlugin extends Plugin
 {
     public static function getExtendedHandlers(): iterable
     {
-        // TODO: Implement getExtendedHandlers() method.
+        yield BlockDefinitionHandlerInterface::class;
     }
 }
