diff --git a/templates/base.html.twig b/templates/base.html.twig
index 3c28c6a..0defaa4 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -16,6 +16,7 @@
 
     <body>
     {% block layout %}
+        {% block nav %}
         <nav class="navbar navbar-expand-lg navbar-light bg-light">
             <a class="navbar-brand" href="{{ path('app_homepage')}}">
                 <img src="{{ asset('images/B-B-logo.png') }}" width="30" height="30" alt="bark and bake logo">
@@ -44,9 +45,11 @@
                 </form>
             </div>
         </nav>
+        {% endblock %}
 
         {% block body %}{% endblock %}
 
+        {% block footer %}
         <div class="container mt-5">
             <footer class="d-flex flex-wrap justify-content-between align-items-center py-3 my-4 border-top">
                 <div class="col-md-6 d-flex align-items-center">
@@ -59,6 +62,7 @@
                 </div>
             </footer>
         </div>
+        {% endblock %}
     {% endblock %}
     </body>
 </html>
