diff --git a/src/Layouts/VerticalWhitespacePlugin.php b/src/Layouts/VerticalWhitespacePlugin.php
new file mode 100644
index 0000000..fcf273d
--- /dev/null
+++ b/src/Layouts/VerticalWhitespacePlugin.php
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Layouts;
+
+use Netgen\Layouts\Block\BlockDefinition\Handler\Plugin;
+
+class VerticalWhitespacePlugin extends Plugin
+{
+    public static function getExtendedHandlers(): iterable
+    {
+        // TODO: Implement getExtendedHandlers() method.
+    }
+}
