diff --git a/src/ContentBrowser/RecipeBrowserItem.php b/src/ContentBrowser/RecipeBrowserItem.php
index 965e36e..36e78b7 100644
--- a/src/ContentBrowser/RecipeBrowserItem.php
+++ b/src/ContentBrowser/RecipeBrowserItem.php
@@ -30,4 +30,9 @@ class RecipeBrowserItem implements ItemInterface
     {
         return true;
     }
+
+    public function getRecipe(): Recipe
+    {
+        return $this->recipe;
+    }
 }
