diff --git a/.env b/.env
index 30eaad2..4621adf 100644
--- a/.env
+++ b/.env
@@ -29,6 +29,6 @@ DATABASE_URL="mysql://app:!ChangeMe!@127.0.0.1:3306/app?serverVersion=8&charset=
 ###< doctrine/doctrine-bundle ###
 
 ###> contentful/contentful-bundle ###
-CONTENTFUL_SPACE_ID=cfexampleapi
-CONTENTFUL_ACCESS_TOKEN=b4c0n73n7fu1
+CONTENTFUL_SPACE_ID=uvx9svgj8l12
+CONTENTFUL_ACCESS_TOKEN=3qgirZC8zMKQEnGgXNtrjRibdXYuhiFEBY9tHPyfjnw
 ###< contentful/contentful-bundle ###
