diff --git a/templates/base.html.twig b/templates/base.html.twig
index 9296dfd..e3c7de8 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -44,7 +44,9 @@
             </div>
         </nav>
 
+        {% block layout %}
         {% block body %}{% endblock %}
+        {% endblock %}
 
         <div class="container mt-5">
             <footer class="d-flex flex-wrap justify-content-between align-items-center py-3 my-4 border-top">
