diff --git a/templates/base.html.twig b/templates/base.html.twig
index aa31ff4..ea5bad4 100644
--- a/templates/base.html.twig
+++ b/templates/base.html.twig
@@ -15,6 +15,7 @@
     </head>
 
     <body>
+    {% block layout %}
         <nav class="navbar navbar-expand-lg navbar-light bg-light">
             <a class="navbar-brand" href="{{ path('app_homepage')}}">
                 <img src="{{ asset('images/B-B-logo.png') }}" width="30" height="30" alt="bark and bake logo">
@@ -58,5 +59,6 @@
                 </div>
             </footer>
         </div>
+    {% endblock %}
     </body>
 </html>
