diff --git a/templates/recipes/list.html.twig b/templates/recipes/list.html.twig
index 593d1b0..e71b51e 100644
--- a/templates/recipes/list.html.twig
+++ b/templates/recipes/list.html.twig
@@ -1,4 +1,4 @@
-{% extends 'base.html.twig' %}
+{% extends nglayouts.layoutTemplate %}
 
 {% block title %}All Recipes{% endblock %}
 
diff --git a/templates/recipes/show.html.twig b/templates/recipes/show.html.twig
index 9c17886..4e02f4e 100644
--- a/templates/recipes/show.html.twig
+++ b/templates/recipes/show.html.twig
@@ -1,4 +1,4 @@
-{% extends 'base.html.twig' %}
+{% extends nglayouts.layoutTemplate %}
 
 {% block title %}{{ recipe.name }}{% endblock %}
 
diff --git a/templates/security/login.html.twig b/templates/security/login.html.twig
index a2ad57e..f609fe0 100644
--- a/templates/security/login.html.twig
+++ b/templates/security/login.html.twig
@@ -1,4 +1,4 @@
-{% extends 'base.html.twig' %}
+{% extends nglayouts.layoutTemplate %}
 
 {% block title %}Log in!{% endblock %}
 
