diff --git a/src/AppExtension.php b/src/AppExtension.php
new file mode 100644
index 0000000..08c87a3
--- /dev/null
+++ b/src/AppExtension.php
@@ -0,0 +1,24 @@
+<?php
+
+namespace App;
+
+use Symfony\Component\Config\Resource\FileResource;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\Extension;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
+use Symfony\Component\Yaml\Yaml;
+
+class AppExtension extends Extension implements PrependExtensionInterface
+{
+    public function load(array $configs, ContainerBuilder $container)
+    {
+    }
+
+    public function prepend(ContainerBuilder $container)
+    {
+        $configFile = __DIR__ . '/../config/prepends/netgen_layouts.yaml';
+        $config = Yaml::parse((string) file_get_contents($configFile));
+        $container->prependExtensionConfig('netgen_layouts', $config['netgen_layouts']);
+        $container->addResource(new FileResource($configFile));
+    }
+}
diff --git a/src/Kernel.php b/src/Kernel.php
index 779cd1f..671014b 100644
--- a/src/Kernel.php
+++ b/src/Kernel.php
@@ -3,9 +3,19 @@
 namespace App;
 
 use Symfony\Bundle\FrameworkBundle\Kernel\MicroKernelTrait;
+use Symfony\Component\Config\Loader\LoaderInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 use Symfony\Component\HttpKernel\Kernel as BaseKernel;
 
 class Kernel extends BaseKernel
 {
-    use MicroKernelTrait;
+    use MicroKernelTrait { configureContainer as baseConfigureContainer; }
+
+    private function configureContainer(ContainerConfigurator $container, LoaderInterface $loader, ContainerBuilder $builder): void
+    {
+        $this->baseConfigureContainer($container, $loader, $builder);
+
+        $builder->registerExtension(new AppExtension());
+    }
 }
