diff --git a/src/ContentBrowser/BrowserRootLocation.php b/src/ContentBrowser/BrowserRootLocation.php
new file mode 100644
index 0000000..5619956
--- /dev/null
+++ b/src/ContentBrowser/BrowserRootLocation.php
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\ContentBrowser;
+
+use Netgen\ContentBrowser\Item\LocationInterface;
+
+class BrowserRootLocation implements LocationInterface
+{
+    public function getLocationId()
+    {
+        // TODO: Implement getLocationId() method.
+    }
+
+    public function getName(): string
+    {
+        // TODO: Implement getName() method.
+    }
+
+    public function getParentId()
+    {
+        // TODO: Implement getParentId() method.
+    }
+}
