diff --git a/src/Layouts/LatestRecipeQueryTypeHandler.php b/src/Layouts/LatestRecipeQueryTypeHandler.php
index a310741..f811672 100644
--- a/src/Layouts/LatestRecipeQueryTypeHandler.php
+++ b/src/Layouts/LatestRecipeQueryTypeHandler.php
@@ -23,7 +23,7 @@ class LatestRecipeQueryTypeHandler implements QueryTypeHandlerInterface
 
     public function getValues(Query $query, int $offset = 0, ?int $limit = null): iterable
     {
-        return $this->recipeRepository->createQueryBuilderOrderedByNewest()
+        return $this->recipeRepository->createQueryBuilderOrderedByNewest($query->getParameter('term')->getValue())
             ->setFirstResult($offset)
             ->setMaxResults($limit)
             ->getQuery()
@@ -32,7 +32,7 @@ class LatestRecipeQueryTypeHandler implements QueryTypeHandlerInterface
 
     public function getCount(Query $query): int
     {
-        return $this->recipeRepository->createQueryBuilderOrderedByNewest()
+        return $this->recipeRepository->createQueryBuilderOrderedByNewest($query->getParameter('term')->getValue())
             ->select('COUNT(recipe.id)')
             ->getQuery()
             ->getSingleScalarResult();
