diff --git a/src/Layouts/LatestRecipeQueryTypeHandler.php b/src/Layouts/LatestRecipeQueryTypeHandler.php
index b370061..a310741 100644
--- a/src/Layouts/LatestRecipeQueryTypeHandler.php
+++ b/src/Layouts/LatestRecipeQueryTypeHandler.php
@@ -6,6 +6,7 @@ use App\Repository\RecipeRepository;
 use Netgen\Layouts\API\Values\Collection\Query;
 use Netgen\Layouts\Collection\QueryType\QueryTypeHandlerInterface;
 use Netgen\Layouts\Parameters\ParameterBuilderInterface;
+use Netgen\Layouts\Parameters\ParameterType\TextType;
 use Symfony\Component\DependencyInjection\Attribute\AutoconfigureTag;
 
 #[AutoconfigureTag('netgen_layouts.query_type_handler', ['type' => 'latest_recipes'])]
@@ -17,6 +18,7 @@ class LatestRecipeQueryTypeHandler implements QueryTypeHandlerInterface
 
     public function buildParameters(ParameterBuilderInterface $builder): void
     {
+        $builder->add('term', TextType::class);
     }
 
     public function getValues(Query $query, int $offset = 0, ?int $limit = null): iterable
