diff --git a/src/Layouts/ContentfulSlugger.php b/src/Layouts/ContentfulSlugger.php
new file mode 100644
index 0000000..a959403
--- /dev/null
+++ b/src/Layouts/ContentfulSlugger.php
@@ -0,0 +1,14 @@
+<?php
+
+namespace App\Layouts;
+
+use Netgen\Layouts\Contentful\Entity\ContentfulEntry;
+use Netgen\Layouts\Contentful\Routing\EntrySluggerInterface;
+
+class ContentfulSlugger implements EntrySluggerInterface
+{
+    public function getSlug(ContentfulEntry $contentfulEntry): string
+    {
+        // TODO: Implement getSlug() method.
+    }
+}
