diff --git a/src/Layouts/ContentfulSlugger.php b/src/Layouts/ContentfulSlugger.php
index 9068486..2d97b72 100644
--- a/src/Layouts/ContentfulSlugger.php
+++ b/src/Layouts/ContentfulSlugger.php
@@ -5,7 +5,9 @@ namespace App\Layouts;
 use Netgen\Layouts\Contentful\Entity\ContentfulEntry;
 use Netgen\Layouts\Contentful\Routing\EntrySlugger\FilterSlugTrait;
 use Netgen\Layouts\Contentful\Routing\EntrySluggerInterface;
+use Symfony\Component\DependencyInjection\Attribute\AutoconfigureTag;
 
+#[AutoconfigureTag('netgen_layouts.contentful.entry_slugger', ['type' => 'default_slugger'])]
 class ContentfulSlugger implements EntrySluggerInterface
 {
     use FilterSlugTrait;
