diff --git a/templates/nglayouts/themes/standard/block/title.html.twig b/templates/nglayouts/themes/standard/block/title.html.twig
index 28a790d..a417f27 100644
--- a/templates/nglayouts/themes/standard/block/title.html.twig
+++ b/templates/nglayouts/themes/standard/block/title.html.twig
@@ -9,7 +9,7 @@
     {# Located inside the "content" block to include the context from the parent template #}
     {% set title = macros.inline_template(block.parameter('title').value, _context) %}
 
-    <{{ tag }} class="title">
+    <{{ tag }}>
         {% if block.parameter('use_link').value and not link.empty %}
             {{ nglayouts_render_parameter(link, {content: title}) }}
         {% else %}
