diff --git a/templates/nglayouts/themes/standard/block/contentful_entry_field/assets.html.twig b/templates/nglayouts/themes/standard/block/contentful_entry_field/assets.html.twig
index 11f654f..6f4f1ba 100644
--- a/templates/nglayouts/themes/standard/block/contentful_entry_field/assets.html.twig
+++ b/templates/nglayouts/themes/standard/block/contentful_entry_field/assets.html.twig
@@ -2,6 +2,5 @@
 
 {% block content %}
     {% set field = block.dynamicParameter('field') %}
-    {{ dump() }}
-    <img src="{{ field.value.file.url }}?h={{ block.parameter('height').value }}&w={{ block.parameter('width').value }}" alt="{{ field.value.title }}" />
+    <img class="{{ css_class }}" src="{{ field.value.file.url }}?h={{ block.parameter('height').value }}&w={{ block.parameter('width').value }}" alt="{{ field.value.title }}" />
 {% endblock %}
